app/models/product.rb

```diff
@@ -12,7 +12,7 @@ class Product < ApplicationRecord
   validates :price, numericality: { only_float: true }
   #may need to include integers as well
   validates :merchant_id, presence: true
-
+  # TODO: Add a validation for quanty
```

Comment:
Idea for validation:

`validates :quantity, presence: true, numericality: {only_integer: true, greater_than_or_equal_to: 0 }`